server/src/test/java/io/crate/expression/scalar/DateTruncFunctionTest.java

```diff
@@ -52,17 +54,17 @@ public void test_date_trunc_works_with_timestamp_without_timezone() {
     }
 
     @Test
-    public void testNormalizeNullInterval() throws Exception {
+    public void testNormalizeNullInterval() {
         assertNormalize("date_trunc(null, 919946281123)", isLiteral(null));
     }
 
     @Test
-    public void testEvaluateNullInterval() throws Exception {
+    public void testEvaluateNullInterval() {
         assertEvaluate("date_trunc(interval, 919946281123)", null, Literal.of((String) null));
     }
 
     @Test
-    public void testInvalidInterval() throws Exception {
+    public void testInvalidInterval() {
```

Comment:
Is this forced by checkstyle? Would be good to keep those `throws Exception` clauses in tests. It gets really annoying if we have to add/remove/add whenever we make some changes.

Reply:
No it's not, sorry that was a fix-warnings I did manually on IDE, I will revert.